src/main/scala/io/sdkman/changelogs/SchemaCrawlerMigration.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "071")
+class SchemaCrawlerMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_schemacrawler",
+    author = "sualeh"
+  )
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "schemacrawler",
+      name = "SchemaCrawler",
+      description =
+        "SchemaCrawler is a free database schema discovery and comprehension tool",
```

Comment:
This will appear on our website under _SDKs_, as well as when you run the `sdk list` command. Could you add a bit more to make it a short paragraph?

Reply:
Sure. Done.